test/common/http/conn_manager_utility_test.cc

```diff
@@ -623,6 +671,8 @@ TEST_F(ConnectionManagerUtilityTest, NonTlsAlwaysForwardClientCert) {
       .WillByDefault(Return(Http::ForwardClientCertType::AlwaysForwardOnly));
   std::vector<Http::ClientCertDetailsType> details;
   ON_CALL(config_, setCurrentClientCertDetails()).WillByDefault(ReturnRef(details));
+  const std::string generated_uuid = "f4dca0a9-12c7-4307-8002-969403baf480";
```

Comment:
I would move the UUID and the ON_CALL default into the test fixture constructor, which will avoid a lot of duplication. You could then just EXPECT_CALL(); in tests if still want to have a real expect in the tests that do that (it will still use the default action).